addons/account/views/report_invoice.xml

```diff
@@ -33,6 +33,7 @@
                                     <t t-if="o.company_id.account_fiscal_country_id.vat_label" t-out="o.company_id.account_fiscal_country_id.vat_label" id="inv_tax_id_label"/>
                                     <t t-else="">Tax ID</t>: <span t-field="o.partner_id.vat"/>
                                 </div>
+                                <t t-if="o.partner_id.country_code == 'MA'">ICE: <span t-field="o.partner_id.l10n_ma_ice"/></t>
```

Comment:
you can't do that: `o.partner_id.l10n_ma`_ice is defined in `l10n_ma`, not in `account`, and the code that adds the field in the xml must be in `l10n_ma` also

Reply:
wait a minute, the field isn't named the same :facepalm: 

It's good practice to read your diff before asking a review ;)

Reply:
The one defined in the `res.company` is the one of the company we have. and `l10n_ma_ice` is the one defined in the `res.partner` table and it falls under `l10n_ma_report` module. So, the name of the field is actually right!

for the second point, I think i got you, we need to add this field in the `l10n_ma_report` module where the field is defined, is it?

Reply:
> for the second point, I think i got you, we need to add this field in the `l10n_ma_report` module where the field is defined, is it?

yes